app/views/legislation/questions/_votes.html.erb

```diff
@@ -0,0 +1,79 @@
+<% voted_classes = css_classes_for_vote(@legislation_question_votes, @question) %>
+<div id="<%= dom_id(@question)%>_votes" class="votes">
+  <% if user_signed_in? && current_user.level_two_verified? %>
+    <span class="in-favor">
+      <% if can?(:vote, question) %>
+        <%= link_to vote_legislation_process_question_path(question.process, question, value: "yes"),
+          method: "post", remote: true, class: "like #{voted_classes[:in_favor]}", title: t("legislation.questions.show.clear") do %>
+          <span class="icon-like"></span>
+          <span class="text"><%= t("legislation.questions.show.clear") %></span>
+        <% end %>
+      <% else %>
+        <%= link_to new_user_session_path do %>
```

Comment:
I think there's a typo here :thinking:. There's a link to whe `new_user_session_path`, but we're in the scenario where the user is already identified :thinking:. In the "against" part of this scenario, this element is not a link.